vanilla-framework/vanilla-brochure-theme
# Bump vf

## Done

Bump VF version to latest
Replace @extend with @include

## QA

Run gulp jekyll and head to http://127.0.0.1:4001 and make sure everything looks okay
